server/publications/collections/products.js

```diff
@@ -308,47 +341,24 @@ Meteor.publish("Products", function (productScrollLimit = 24, productFilters, so
 
   // TODO: Consider publishing the non-admin publication if a user is not in "edit mode" to see what is published
 
-  // Active shop
-  const shopId = Reaction.getShopId();
-  const primaryShopId = Reaction.getPrimaryShopId();
-
-  // Get a list of shopIds that this user has "createProduct" permissions for (owner permission is checked by default)
-  const userAdminShopIds = Reaction.getShopsWithRoles(["createProduct"], this.userId);
-
-  // Don't publish if we're missing an active or primary shopId
-  if (!shopId || !primaryShopId) {
-    return this.ready();
-  }
-
-  // Get active shop id's to use for filtering
-  const activeShopsIds = Shops.find({
```

Comment:
moved this work up to the `filterProducts` methods so that it can be shared by both the `"Products"` and `"Products/grid"` publications.